src/components/admin/donations/grid/Grid.tsx

```diff
@@ -154,7 +154,7 @@ export default observer(function Grid() {
       width: 120,
       resizable: false,
       renderCell: (params: GridRenderCellParams) => {
-        return params.row?.status === 'succeeded' ? (
+        return params.row?.status === 'succeeded' && params.row?.provider === 'stripe' ? (
```

Comment:
Could you please use the enums for the checks:
DonationStatus.succeeded
PaymentProvider.stripe